junit5-engine/src/main/java/org/junit/gen5/engine/junit5/descriptor/MethodTestDescriptor.java

```diff
@@ -175,17 +175,17 @@ private void invokeBeforeTestMethodCallbacks(ExtensionRegistry registry, TestExt
 				.forEach(extension -> executeAndMaskThrowable(() -> extension.beforeTestMethod(context)));
 	}
 
-	private void invokeTestMethod(ExtensionRegistry ExtensionRegistry, TestExtensionContext testExtensionContext,
+	protected void invokeTestMethod(JUnit5EngineExecutionContext context, TestExtensionContext testExtensionContext,
```

Comment:
Actually, it looks like this method should not have been modified at all. The `context` is not used except to access the `ExtensionRegistry`. Thus, the changes look superfluous, perhaps simply due to multiple merges with master over time.